OHHTTPStubs/UnitTests/Test Suites/SwiftHelpersTests.swift

```diff
@@ -52,6 +52,31 @@ class SwiftHelpersTests : XCTestCase {
     }
   }
 
+  func testIsAbsoluteURLString() {
+    let matcher = isAbsoluteURLString("foo://foo/bar?param1=123&param2=foo#anchor")
+
+    let urls = [
+      "foo:": false,
+      "foo://foo/bar/baz": false,
+      "foo://foo/bar?param1=123&param2=foo#anchor": true,
+      "foo://foo/bar?param1=123&param2=foo": false,
+      "foo://foo/bar?param1=123": false,
+      "foo://foo/bar": false,
+      "bar://foo/bar": false,
+      "bar://foo/ba": false,
+      "foobar://": false
+    ]
+
+    for (url, result) in urls {
+      #if swift(>=3.0)
+        let req = URLRequest(url: URL(string: url)!)
+      #else
+        let req = NSURLRequest(URL: NSURL(string: url)!)
```

Comment:
Unrelated to this PR directly, but I see we have that pattern on multiple tests: maybe we should add a `func request(from string: String) -> URLRequest` helper function for our tests, responsible for handling Swift 2/3 compatibility, and use that in our test cases to avoid repeating those `#if Swift(>=3.0)` everywhere? (Might be a separate PR though)